account_invoice_validate_send_email/models/account_invoice.py

```diff
@@ -45,8 +46,8 @@ def get_mail_compose_message(self):
             {
                 "default_model": "account.invoice",
                 "default_res_id": self.ids[0],
-                "default_use_template": bool(template_id),
-                "default_template_id": template_id,
+                "default_use_template": bool(template.id),
```

Comment:
```suggestion
                "default_use_template": bool(template),
```
I suppose this should do.